networking_cisco/plugins/cisco/cfg_agent/device_drivers/asr1k/aci_asr1k_snippets.py

```diff
@@ -154,3 +154,27 @@
         </cli-config-data>
 </config>
 """
+
+GLOBAL_CONFIG_PREFIX = """
```

Comment:
I'm not sure why we need to separate PREFIX and POSTFIX here.....It seems to me we can just put them into one command like this:
 SET_GLOBAL_CONFIG = """
 <config>
         <cli-config-data>
             <cmd>%s</cmd>
         </cli-config-data>
 </config>
 """